test/hotspot/jtreg/compiler/vectorization/runner/ArrayInvariantFillTest.java

```diff
@@ -50,6 +50,7 @@ public class ArrayInvariantFillTest extends VectorizationTestRunner {
 
     private static final int SIZE = 543;
 
+    private boolean booleanInv;
```

Comment:
Looks like this field is not used anywhere here.

Reply:
Thanks, I will fix it.